<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Bus</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Parameters</string>
    </attribute>
    <widget class="QLabel" name="vn_kv_label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>14</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>vn_kv</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="vn_kv">
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>12</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>40</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="name">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Documentation</string>
    </attribute>
    <widget class="QWebView" name="webView_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>0</y>
       <width>201</width>
       <height>391</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>http://pandapower.readthedocs.io/en/v1.3.0/_images/bus.png</string>
      </url>
     </property>
    </widget>
    <widget class="QWebView" name="webView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>381</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>http://pandapower.readthedocs.io/en/v1.3.0/elements/bus.html#bus</string>
      </url>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>429</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>429</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
